Homework4

anenvs opened the pull request:
Homework4

anenvs committed:
Домашняя работа. Урок 2

anenvs committed:
Домашняя работа. Урок 3

anenvs committed:
Домашняя работа. Урок 4

FlameXander commented on src/main/java/ru/otus/homework4/Box.java:
```diff
@@ -0,0 +1,86 @@
+        }
+    }
+
+    public boolean getIsOpen() {
```
Геттер обычно ничего кроме возврата значения не делает, поэтому распечатки тут лишние

FlameXander commented on src/main/java/ru/otus/homework4/Box.java:
```diff
@@ -0,0 +1,86 @@
+    public Box(String color, String size) {
+        this.color = color;
+        this.size = size;
+        isOpen = true;
```
Для единообразия и "красоты" кода можно this. дописать и для остальных двух полей

FlameXander commented on src/main/java/ru/otus/homework4/Box.java:
```diff
@@ -0,0 +1,86 @@
+    private String size;
+    private boolean isOpen;
+    private boolean isEmpty;
+    private String Item;
```
Почему имя поля не в camelCase?

FlameXander commented on src/main/java/ru/otus/homework4/Box.java:
```diff
@@ -0,0 +1,86 @@
+        this.isOpen = isOpen;
+    }
+
+    public void fillBox(String Item) {
```
Почему имя аргумента не в camelCase?

FlameXander commented on src/main/java/ru/otus/homework4/User.java:
```diff
@@ -0,0 +1,68 @@
+        return this;
+    }
+
+    private String name;
```
Обычно поля в самом верху класса объявляют (но тут по желанию)

FlameXander commented on src/main/java/ru/otus/homework4/Box.java:
```diff
@@ -0,0 +1,86 @@
+        return isOpen;
+    }
+
+    public void setIsOpen(boolean isOpen) {
```
Лучше заменить на пару методов - open() и close(), будет чуть лучше смотреться

anenvs committed:
Домашняя работа. Урок 4
Доработки по замечаниям.
setIsOpen заменен на два отдельных метода open close.
Переименована переменная в camelCase
Косметические исправления
